shared/src/ledger/storage/mod.rs

```diff
@@ -27,6 +33,11 @@ use crate::types::storage::{
     BlockHash, BlockHeight, DbKeySeg, Epoch, Epochs, Key, BLOCK_HASH_LENGTH,
 };
 use crate::types::time::DateTimeUtc;
+#[cfg(feature = "ferveo-tpke")]
+use crate::types::transaction::WrapperTx;
+// TODO: What a miserable hack. But it's soooo much less typing
+#[cfg(not(feature = "ferveo-tpke"))]
+type WrapperTx = ();
```

Comment:
could we not use the conditional compilation on the `pub wrapper_txs` fields instead?

Reply:
As per our previous conversation, I think I am not going to persist the wrappers in the storage at all. Instead, they will be flushed to a separate file on ledger shutdown/crash. This will remove that line